### ISSUE TITLE
Failed to fetch error with local file

### ISSUE THREAD

**Issue by robintw** [radiantearth/stac-browser#100]
I'm trying to use stac-browser for the first time. I have a local catalog.json file, which points to one item JSON file.

Whatever I do, I seem to get a `Failed to fetch` error when loading the stac-browser.

I've tried running it in various ways:
- `npm start -- --open --CATALOG_URL="C:\Users\rwilson3\Documents\code\test-stac\catalog.json"` - with a catalog URL set to the full (Windows) path to my catalog.json file
- `npm start -- --open --CATALOG_URL="file:///C:/Users/rwilson3/Documents/code/test-stac/catalog.json"` - the same, but with an explicit `file://` path
- `npm start -- --open --CATALOG_URL="http://localhost:8000/catalog.json"` - with a localhost URL, and the files in the `test-stac` folder being served by the standard Python HTTP server (by running`python` -m http.server` in the folder)

With all of these examples, I can access the JSON file from the browser using the URL given as `CATALOG_URL`.

What am I doing wrong here? Is stac-browser expected to work with local JSON files like this?

**Comment by robintw** [comment]
Ah, I've got it working with the `http://localhost:8000/catalog.json` URL, by using a version of the Python HTTP server that deals with CORS properly. Sorry about that.

However, my other question remains: is it possible to run with just a local file?

**Comment by m-mohr** [comment]
No, it's not possible to run this with local files due to the simple fact that JavaScript in the browser is not allowed to access the local file system for security reasons. There's nothing we can do about it. You always need to spin up a server that serves your local files via HTTP(S).

We may simplify this in the future via CLI (i.e. spin up a web server in the background automatically, see #68).

Great that you were able to make it work with the Python server. We may want to add a note to the README explaining how to run local files though. Happy to accept PRs.
